app/views/account/AccountSettingsView.coffee

```diff
@@ -83,6 +84,12 @@ module.exports = class AccountSettingsView extends CocoView
       confirmModal.on 'confirm', @resetProgress
       @openModalView confirmModal
 
+  onClickResendVerificationEmail: (e) ->
+    $.post "/db/user/#{me.id}/request-verify-email", ->
+      link = $(e.currentTarget)
+      link.find('.resend-text').addClass('hide')
+      link.find('.sent-text').removeClass('hide')
```

Comment:
Is this a new policy, to not put URLs in views at all? I missed that–do we have any more info on it?